boogiestats/boogie_ui/views.py

```diff
@@ -235,6 +235,8 @@ def get_context_data(self, **kwargs):
         if hasattr(self.request.user, "player"):
             context["is_rival"] = self.request.user.player.rivals.filter(id=player_id).exists()
 
+        context["wrapped_years"] = [x for x in range(player.join_date.year, today.year + 1)]
```

Comment:
change to `list(range(...))` or just  `range(...)` if it works. This list comprehension is a leftover from some dropped idea